### PR TITLE
[CARBONDATA-1375] clean hive pom

### PR DESCRIPTION
**Pull request by cenyuhai**
clean hive pom

### CODE REVIEW

**Comment by cenyuhai** [comment]
retest please

**Comment by ravipesala** [comment]
SDV Build Success with Spark 2.1, Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/193/

**Comment by chenliang613** [comment]
retest this please

**Comment by chenliang613** [comment]
@cenyuhai  can you explain this PR, which part you removed, which part you added, and why ?
Just i reviewed, only some pom script be changed the position, very less changed.

**Comment by cenyuhai** [comment]
1.use hadoop.version instead of 2.6.0
2.use hive.version instead of 1.2.1
3.remove thrift
4.remove zookeeper
5.remove spark-hive and spark-sql

**Comment by CarbonDataQA** [comment]
SDV Build Success with Spark 2.1, Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/240/

File: integration/hive/pom.xml [hunk 1]
```diff
@@ -36,122 +36,96 @@
...
-            </exclusions>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-client</artifactId>
+          <version>${hadoop.version}</version>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
```
Comment (cenyuhai) [inline integration/hive/pom.xml]: ok

**Comment by cenyuhai** [comment]
retest please

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**